examples/shopping/carts.rst

```diff
@@ -0,0 +1,107 @@
+.. _shopping-carts:
+
+Shopping Carts
+==============
+
+At this point, we have Captured Collections setup for users and products, and we're ready to
```

Comment:
nit: i haven't been capitalizing like this elsewhere. `Captured Collections` => `captured collections`.

Reasons: a) tough to keep consistent on Capitalization, and b) de-emphasizing the primacy of Collections for users ("it's just a collection, not "The Collection").